docs/examples-more.md

```diff
@@ -0,0 +1,8 @@
+---
+id: examples-more
+title: More...
+---
+
+In this section, we will list a lot of cool examples.
+
+### [Custom form layout with `display: grid;`](examples-layout-display-grid)
```

Comment:
Each example should be prefixed with this page id, e.g. `examples-more-layout-display-grid` (or `examples-random-layout-display-grid`, according to the last comment). Also, how about renaming it to `grid-layout`? It feels better.